change-notes/1.25/analysis-javascript.md

```diff
@@ -23,6 +23,29 @@
 | Expression has no effect (`js/useless-expression`) | Less results | This query no longer flags an expression when that expression is the only content of the containing file. |
 | Unknown directive (`js/unknown-directive`) | Less results | This query no longer flags directives generated by the Babel compiler. |
 
+The following low-precision queries are no longer run by default on LGTM (their results already were not displayed previously):
```

Comment:
Saying both "already" and "previously" seems a bit redundant.

Reply:
I have removed "previously".